pallets/phala/src/fat.rs

```diff
@@ -258,6 +262,30 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::weight(0)]
+		pub fn cluster_set_log_receiver(
+			origin: OriginFor<T>,
+			cluster: ContractClusterId,
+			log_receiver: ContractId,
+		) -> DispatchResult {
+			let deployer = ensure_signed(origin)?;
+			let cluster_info = Clusters::<T>::get(&cluster).ok_or(Error::<T>::ClusterNotFound)?;
+			ensure!(
+				check_cluster_permission::<T>(&deployer, &cluster_info),
```

Comment:
`check_cluster_permission` only checks the permission for deployment. We should consider only allow the owner to set the log receiver.

Reply:
Done